code/config.py

```diff
@@ -1,120 +1,360 @@
-labels_test = 'labels_test.csv'
+# Target number of training patches per class.
+parser.add_argument("--num-train-per-class",
+                    type=int,
+                    default=80000,
+                    help="Target number of training samples per class")
 
-###########################################
-#     PROCESSING AND PATCH GENERATION     #
-###########################################
+# Only looks for purple images and filters whitespace.
+parser.add_argument(
+    "--type-histopath",
+    type=bool,
+    default=True,
+    help="Only look for purple histopathology images and filter whitespace")
 
-# this is the input for model training, automatically built
-train_folder = 'train_folder'
-train_patches = 'train_folder/train'
-val_patches = 'train_folder/val'
+# Sliding window overlap factor (for testing).
+# For generating patches during the training phase, we slide a window to overlap by some factor.
+# Must be an integer. 1 means no overlap, 2 means overlap by 1/2, 3 means overlap by 1/3.
+# Recommend 2 for very high resolution, 3 for medium, and 5 not extremely high resolution images.
+parser.add_argument("--slide-overlap",
+                    type=int,
+                    default=3,
+                    help="Sliding window overlap factor for the testing phase")
 
-# folders of patches by wsi in training set, used for getting train acc on wsi leve;
-patches_eval_train = 'patches_eval_train'
+parser.add_argument("--image-ext",
+                    type=str,
+                    default="jpg",
+                    help="Image extension for saving patches")
 
-# folders of patches by wsi in val set, used for getting val acc on wsi level
-patches_eval_val = 'patches_eval_val'
+#########################################
+#               TRANSFORM               #
+#########################################
+parser.add_argument(
+    "--color-jitter-brightness",
+    type=float,
+    default=0.5,
+    help=
+    "Random brightness jitter to use in data augmentation for ColorJitter() transform"
+)
+parser.add_argument(
+    "--color-jitter-contrast",
+    type=float,
+    default=0.5,
+    help=
+    "Random contrast jitter to use in data augmentation for ColorJitter() transform"
+)
+parser.add_argument(
+    "--color-jitter-saturation",
+    type=float,
+    default=0.5,
+    help=
+    "Random saturation jitter to use in data augmentation for ColorJitter() transform"
+)
+parser.add_argument(
+    "--color-jitter-hue",
+    type=float,
+    default=0.2,
+    help=
+    "Random hue jitter to use in data augmentation for ColorJitter() transform"
+)
 
-# folders of patches by wsi in test set, used for getting test acc on wsi level
-patches_eval_test = 'patches_eval_test'
+########################################
+#               TRAINING               #
+########################################
+# Model hyperparameters.
+parser.add_argument("--num-epochs",
+                    type=int,
+                    default=10,
```

Comment:
seems small to me

Reply:
Once again, this was because I was testing.  I should changed it back.

Reply:
Changed back in newest commit.